lib/core.dart

```diff
@@ -10,6 +10,7 @@
 
```

Comment:
Regarding the API question: I was wondering if we could also export `dcaf` as a whole, making it available for all library users when using its classes in public APIs, or if this is considered bad practice.

```suggestion
export 'package:dcaf/dcaf.dart';
```

Reply:
AFAICT, *Effective Dart* doesn't mention anything about exporting other libraries, so it's probably fine. I think this is a good idea anyway since it's so tightly coupled, so feel free to export `dcaf`.

Reply:
Great :) Then I'll add our original approach back in, directly using the `dcaf` class for passing both the creation hint and the access token.